js/main.js

```diff
@@ -329,6 +329,25 @@ define([
             }));
           }
           registry.byId("mainWindow").layout();
+          
+          // *******************************************
+          // **** Maptiks Changes below
+          // *******************************************
+          
+          require(['maptiks'], function (mapWrapper) {
+            var container = response.map.container;
+            var maptiksMapOptions = {
+                extent: response.map.extent,
+                maptiks_trackcode: "c311cf16-ad79-42b1-97f9-f433be6c8b00",
+                maptiks_id: "basictest"
```

Comment:
If there is no UI for the user to enter these, then let's use this:
`maptiks_trackcode: "ENTER_YOUR_TRACKCODE_HERE", `
`maptiks_id: "ENTER_YOUR_MAPNAME_HERE", `